Packs/Zscaler/Integrations/Zscaler/Zscaler.py

```diff
@@ -1016,6 +1015,106 @@ def set_user_command(args):
         return responseJson
 
 
+def get_ip_destination_groups_lite_command(args):
+    exclude_type = str(args.get('exclude_type', '')).strip()
+    category_type = argToList(args.get('type', []))
+    include_ipv6 = argToBoolean(args.get('include_ipv6', False))
+    if include_ipv6:
+        try:
+            ipv4_cmd_url = "/ipDestinationGroups/lite"
+            ipv6_cmd_url = f"/ipDestinationGroups/ipv6DestinationGroups/lite"
+            if exclude_type != '':
+                ipv4_cmd_url += f"?excludeType={exclude_type}"
+                ipv6_cmd_url += f"?excludeType={exclude_type}"
+            else:
+                for i in range(len(category_type)):
+                    if i == 0:
+                        ipv4_cmd_url += f"?type={category_type[i]}"
+                        ipv6_cmd_url += f"?type={category_type[i]}"
+                    else:
+                        ipv4_cmd_url += f"&type={category_type[i]}"
+                        ipv6_cmd_url += f"&type={category_type[i]}"
+
+            ipv4_responses = http_request('GET', ipv4_cmd_url).json()
+            ipv6_responses = http_request('GET', ipv6_cmd_url).json()
```

Comment:
Only these statements can result in `JSONDecodeError`, therefore these should only be inside the `try` block. Previous statements are not required to be under `try` block.

💡  Good practice is to keep the try block as small as possible: It's best to keep only the code that might raise an exception inside the try block. This makes it easier to locate the source of the error when an exception is raised.